test-suite/metaschema-xspec/json-schema-gen/json-value-testing-mini-metaschema-COMPOSED.xml

```diff
@@ -0,0 +1,244 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should be generated by the build and as a derived artifact should not be included in the repo.